src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -227,8 +227,8 @@ def load_arguments(self, _):
             c.argument('settings', nargs='+', help="space-separated app settings in a format of `<name>=<value>`")
             c.argument('setting_names', nargs='+', help="space-separated app setting names")
         with self.argument_context(scope + ' config ssl import') as c:
-            c.argument('key_vault', help='The name or resource ID of the Key Vault')
-            c.argument('key_vault_certificate_name', help='The name of the certificate in Key Vault')
+            c.argument('certificate_name', help='The name of the certificate to be imported')
+            c.argument('key_vault', help='the keyvault id')
```

Comment:
Can it be a name?

Reply:
In this case, name is used for the webapp name.

Reply:
@qwordy could you take another look, I made some changes to avoid it being breaking